sky/packages/sky/lib/sky_tool

```diff
@@ -254,6 +284,355 @@ class StopSky(object):
         pids.clear()
 
 
+class IOSDevice(object):
+    _has_ios_deploy = None
+    @classmethod
```

Comment:
I probably would have just made these all instance methods even if they don't hold any state and then just newed up an IODevice once at the top of your main.  Class methods in python can be annoying to work with.